data/core/faqs.md

```diff
@@ -748,28 +339,6 @@
     - utter_duckling
     - utter_anything_else
 
-## how to get started without privacy policy
-* how_to_get_started
-    - utter_getstarted
-    - utter_first_bot_with_rasa
-* ask_faq_platform OR ask_faq_languages OR ask_faq_tutorialcore OR ask_faq_tutorialnlu OR ask_faq_opensource OR ask_faq_voice OR ask_faq_slots OR ask_faq_channels OR ask_faq_differencecorenlu OR ask_faq_python_version OR ask_faq_community_size OR ask_faq_what_is_forum OR ask_faq_tutorials
-    - action_faqs
-    - utter_first_bot_with_rasa
-* affirm
-    - action_set_onboarding
-    - slot{"onboarding": true}
-    - utter_built_bot_before
-* affirm
-    - utter_ask_migration
-* deny
-    - utter_explain_stack
-    - utter_stack_details
-    - utter_explain_nlucore
-* affirm OR how_to_get_started{"product":"stack"}
-    - utter_explain_nlu
-    - utter_explain_core
-    - utter_direct_to_step2
-
 ## FAQ - tell more about platform
 * ask_faq_platform
```

Comment:
platform?

Reply:
We were planning to keep the old platform FAQs as they did no harm (and didn't turn up with nothing, for someone who is used to using that term)

Reply:
got it